projects/sbb-esta/angular-business/schematics/rollup.config.js

```diff
@@ -3,7 +3,7 @@ import { existsSync, readdirSync } from 'fs';
 import { join } from 'path';
 import copy from 'rollup-plugin-copy';
 
-const dist = join(__dirname, '../../../../dist/sbb-esta/angular-public/schematics');
+const dist = join(__dirname, '../../../../dist/sbb-esta/angular-business/schematics');
```

Comment:
@kyubisation this was likely a bug in the current version, so no schematics were in angular-business

Reply:
Oops. Might be worth to release a patch release just for that...